parts/kuberneteswindowssetup.ps1

```diff
@@ -175,7 +175,7 @@ Write-KubernetesStartFiles($podCIDR)
 c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json
 "@
 
-    if ($global:KubeBinariesVersion -ne "1.5.3" -and $global:KubeBinariesVersion -ne "1.5.7")
+    if ($global:KubeBinariesVersion -ne "1.5.3" -and $global:KubeBinariesVersion -ne "1.5.7" -and $global:KubeBinariesVersion -lt "1.7.0")

```

Comment:
@martell Is `--image-pull-progress-deadline=20m --cgroups-per-qos=false --enforce-node-allocatable=`"`"` not used by 1.7.0? I think at least `--image-pull-progress-deadline=20m` is needed for downloading windows container image.